src/ooxml/java/org/apache/poi/xslf/usermodel/XSLFTextRun.java

```diff
@@ -586,10 +597,12 @@ void copy(XSLFTextRun r){
 
         PaintStyle srcFontColor = r.getFontColor();
         if(srcFontColor != null && !srcFontColor.equals(getFontColor())){
-            setFontColor(srcFontColor);
+            // setFontColor will throw IllegalArgumentException if this is not checked
+            if (srcFontColor instanceof SolidPaint)
+                setFontColor(srcFontColor);
```

Comment:
I'm not sure about this. Is there a good reason for this class not support all font colours? And if there is a good reason not to accept all font colours, shouldn't we throw an exception if we can't accept the font colour?

Reply:
The only reason is that my branch for correct support of `CTTextRun` as `XDDFTextRun` is not ready for reviews. 

https://github.com/cuali/poi/branches